newsroom/push/views.py

```diff
@@ -47,6 +47,10 @@ async def handle_publish_planning(item):
     plan_id = await publisher.publish_planning_item(item, orig.to_dict() if orig else {})
     event_id = await publisher.publish_planning_into_event(item)
 
+    if item.get("related_events"):
+        for events in item["related_events"]:
+            event_id = await publisher.publish_planning_into_event(item, events["_id"])
```

Comment:
Only if the link is not primary (as that would have already been handled by the `publish_planning_into_event` thats executed straight after `publish_planning_item` call

Reply:
got it